etl/steps/data/garden/irena/2024-10-29/renewable_power_generation_costs.py

```diff
@@ -13,22 +13,22 @@ def run(dest_dir: str) -> None:
     #
     # Load meadow dataset and read its tables.
     ds_meadow = paths.load_dataset("renewable_power_generation_costs")
-    tb = ds_meadow["renewable_power_generation_costs"].reset_index()
-    tb_solar_pv = ds_meadow["solar_photovoltaic_module_prices"]
+    tb = ds_meadow.read_table("renewable_power_generation_costs")
```

Comment:
Any reason for preferring using read_table?

Reply:
`read_table` by default loads the table with the index already reset, which is convenient. However, in the case of solar_pv, since I'm not doing anything to the data, I didn't need to reset. In this case, I could have loaded the data with the usual `ds_meadow["..."]`. However, for consistency, I also loaded it with `read_table`.